config/config.py.template

```diff
@@ -564,6 +564,21 @@ if MLLIB_SPARK_VERSION >= 1.3:
     MLLIB_TESTS += [("gmm", MLLIB_PERF_TEST_RUNNER, SCALE_FACTOR,
         MLLIB_JAVA_OPTS, [ConstantOption("gmm")] + MLLIB_CLUSTERING_TEST_OPTS)]
 
+MLLIB_LDA_TEST_OPTS = MLLIB_COMMON_OPTS + [
+    OptionSet("num-documents", [10000], can_scale=True),
+    OptionSet("num-vocab", [1000], can_scale=False),
+    OptionSet("num-topics", [20], can_scale=False),
+    OptionSet("num-iterations", [20]),
+    OptionSet("document-length", [100])]
+
+if MLLIB_SPARK_VERSION >= 1.3:
```

Comment:
This should require version >= 1.4 as well since the LDA tests are not available for 1.3.